sh_files/Sara_total_bringup.sh

```diff
@@ -292,7 +292,7 @@ rosservice call /wm_play_sound "play:
 
         if ${MAPPING}
         then
-            start_terminal_node "SLAM_NAVIGATION" 'roslaunch sara_launch wm_slam_gmapping.launch'  "$PIDFILENAMEESTOP"
+            start_terminal_node "SLAM_NAVIGATION" 'roslaunch sara_navigation wm_slam_gmapping.launch'  "$PIDFILENAMEESTOP"
```

Comment:
So you moved wm_slam_gmapping.launch to sara_navigation. :+1:

Reply:
oui